### DISCUSSION TITLE
Imgur upload rate limit

### DISCUSSION THREAD

**Discussion by ARGOSkby** [sll552/DiscordBee#159] [Q&A]
Hello :) 

I'd like to know if the upload rate limit resets ? The albums covers doesn't seems to upload anymore (i may have too much songs ahah) 

Also the uploader health is set to False . . .

Thanks !

**Comment by sll552** [comment]
There is a daily and a monthly rate limit, so yes they reset at least once a month. https://api.imgur.com/#limits for more info.
Are you using a custom Imgur ID, if not this could help.
